thrift/server.cc

```diff
@@ -352,6 +352,20 @@ thrift_stats::thrift_stats(thrift_server& server) {
 
         sm::make_derive("served", [&server] { return server.requests_served(); },
                         sm::description("Rate of serving Thrift requests.")),
+        sm::make_gauge("serving", [&server] { return server.requests_serving(); },
+                        sm::description("Number of requests being currently served.")),
```

Comment:
In one counter below you explicitly mentioned that this counter is for the "Thrift transport component", but in this counter and a few others you don't. I think we should try to be more consistent. I would add the mention of Thrift to all the counters - even if the metric's name also includes the word "thrift".

Reply:
Sure, makes sense, will add in v2.